configs/zero123.yaml

```diff
@@ -20,8 +20,8 @@ data: # threestudio/data/image.py -> SingleImageDataModuleConfig
     width: [64, 128, 256]
     batch_size: [12, 4, 2]
     resolution_milestones: [200, 300]
-    eval_height: 256
-    eval_width: 256
+    eval_height: 512
+    eval_width: 512
```

Comment:
Zero123 eval height was 256, is now 512. (this may cause OOM for VRAM-constrained systems)

To save time, I'll add that note in your description.